src/lib/OpenEXRCore/openexr_debug.h

```diff
@@ -12,6 +12,8 @@
 extern "C" {
 #endif
 
+/** @file */
+
```

Comment:
Why did you put the @file markers, if you don't aren't generating per-file docs and you don't have any file-level comments?

Reply:
While searching for why things weren't showing up, I came across this: 

https://www.doxygen.nl/manual/docblocks.html#specialblock

> Attention
> Let's repeat that, because it is often overlooked: to document global objects (functions, typedefs, enum, macros, etc), you must document the file in which they are defined. In other words, there must at least be a
> /*! \file */ 

That said, I think I resolved all the issues that led me to add those lines, and I'm not sure they are, in face, necessary, so I may be misinterpreting that statement.

Reply:
Wow, I didn't know that. I retract my question.

I wonder what counts as "global." Maybe the reason that I've never bumped into this is that I'm working in C++ so everything is technically in a namespace, and thus not global? But for these C bindings, there's no enclosing namespace?